### DISCUSSION TITLE
NC AIO change backup location

### DISCUSSION THREAD

**Discussion by janprive** [nextcloud/all-in-one#1614] [Questions] [answered]
I have an automated borg backup running. How do I change the backup location?

**Comment by mikebakke** [comment]
As referred here by Szaimen - I also have this question as I attempted to move my data directory but noticed that there was no way to change the location of the Borg archive.

**Comment by janprive** [comment, marked as answer]
ok! I got it working!

(fingers crossed)

- remove old borg backups
- check integrity of borg backups in aio interface
- old storage: mount /mnt/borg
- new storage: mount /mnt/borgnew
- rsync all files from to borg to borgnew
- repair ownership on borgnew
- umount both storage
- mount new storage on /mnt/borg
- disconnect old storage
- create backup and check logs if succesfull
- update mastercontainer
- update other containers
- start containers
- all up and running version 25.x.x

**Reply by mikebakke** [reply]
OK - I *think* I see what you did - kept the mount point the same, just mounted the new storage on it. That means in the master console the backup location never changes.... Also the rsync should have maintained the ownerships I thought? It certainly seemed to when I did the copies.

I'll try and see if I can make it work with my target solution, I would just have preferred a "cleaner" solution of being able to amend the location coded in the borg container as I wanted to change the hierarchies.

Will update tomorrow when I've tried this way.
